bin/reth/src/db/mod.rs

```diff
@@ -72,9 +72,12 @@ pub enum Subcommands {
 pub struct ListArgs {
     /// The table name
     table: String, // TODO: Convert to enum
-    /// Where to start iterating
+    /// Skip first N entries
     #[arg(long, short, default_value = "0")]
-    start: usize,
+    skip: usize,
```

Comment:
ah right, skip is the same is start in normal order

Reply:
It was just a simple rename